subsys/bluetooth/host/direction.c

```diff
@@ -862,6 +870,7 @@ int hci_df_prepare_conn_cte_req_failed(struct net_buf *buf,
 
 	if (!atomic_test_bit(conn->flags, BT_CONN_CTE_REQ_ENABLED)) {
 		LOG_ERR("Received conn CTE request notification when CTE REQ disabled");
+		bt_conn_unref(conn);
```

Comment:
Any reason why you use a different style here with no empty line between the unref and the return statement? FWIW, I prefer it like this (no empty line), and it also seems to be the prevalent style elsewhere in this module.

Reply:
The missing newline here was just an oversight :) 

As far as I recall, you were the one who originally reviewed my PRs to add empty lines between expressions and return statements, so I'm surprised to hear that your prefer no empty lines here. 

I can revert it back to not having newlines if that is better, but it's a known issue that we have all these undocumented rules that only occasionally is applied depending on the commit author and reviewer :D

Reply:
> As far as I recall, you were the one who originally reviewed my PRs to add empty lines between expressions and return statements, so I'm surprised to hear that your prefer no empty lines here.

It's not something I have a super strong opinion on, and it probably depends a bit on the context. E.g. small branches it looks better to me with something more compact. Anyway, no need to get into a debate about it - what trumps style opinions always (even the maintainer's opinion) is IMO consistency, and with an inverse relation to the scope (module more important than subsystem, subsystem more important than project, etc). Since the existing code in this c-file has a certain style, I'd just stick to that.

> I can revert it back to not having newlines if that is better, but it's a known issue that we have all these undocumented rules that only occasionally is applied depending on the commit author and reviewer :D

Yeah I agree, and admit that my guidance can at times seem self contradictory. However, I will always yield to the existing style of a module (although it may need separately pointing out if I don't notice it and request something different).

Reply:
Guys I'm sorry I derailed the PR for cosmetic issues, I'm really fine with both. You can revert it to suit johan and I'll approve.

Reply:
Reverted :)